aspnetcore/fundamentals/logging/index.md

```diff
@@ -605,6 +605,10 @@ Navigate to the **Log Streaming** page to view application messages. They're log
 
 ![Azure portal application log streaming](index/_static/azure-log-streaming.png)
 
+## Azure Application Insights trace logging
+
+The [Application Insights](https://azure.microsoft.com/services/application-insights/) SDK is capable of collecting trace telemetry from logs generated via the ASP.NET Core logging infrastructure. For more information, see the [Microsoft/ApplicationInsights-aspnetcore Wiki](https://github.com/Microsoft/ApplicationInsights-aspnetcore/wiki/Logging).
```

Comment:
For more information, see ~the~

Reply:
I'll change it, but I disagree because "Wiki" is a countable noun. "For more information, see Wiki." "For more information, see the Wiki."

Reply:
Acrolinx doesn't complain about either variation. I think it flows better with "the".